config/i18n.json

```
@@ -361,6 +361,7 @@
     "KeyboardShortcuts": "Keyboard Shortcuts",
     "MaterialCounts": "Material Counts",
     "Menu": "Menu",
+    "ProfileAge": "Your Destiny data was last synced with the game state {{age}} ago. Refreshing DIM may get newer data, but Bungie.net may also return cached info.",
```

```suggestion
    "ProfileAge": "Destiny servers last sent updated data {{age}} ago.\nRefreshing from DIM may get newer data, but Bungie.net may also repeat cached information.",
```
- "last synced with" sounded a smidge our fault, because "sync" is an action DIM takes
- for not-knowers of "cached" this may help contextually clarify.

what do you think about a \n between these two sentences? breathing room would be nice *especially* when the two are sandwiching an animated timer